lib/src/main/java/org/team100/lib/config/PIDConstants.java

```diff
@@ -0,0 +1,53 @@
+package org.team100.lib.config;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DriverStation;
+
+/** Uses digital inputs 4 and 5. */
```

Comment:
this comment doesn't make sense to me.

also you could make this a "record" and it would be cleaner